00-prerequisites.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: First Steps in LHCb
+subtitle: Pre-workshop Checklist
+minutes: 10
+---
+> ## Learning Objectives {.objectives}
+>
+> * Check you are ready for the workshop
+
+Please read and try the following steps **before** arriving. For
+some of the steps the solution is easy, but requires waiting for
```

Comment:
This could be clearer. I would remove the "easy" part (what does that matter) and instead emphasise more that if people turn up without these things sorted they will be watching rather than participating.